### PR TITLE
Market & Limit order events

### PR DESCRIPTION
**Pull request by wer1st**

### CODE REVIEW

**Review by stefashkaa — APPROVED** [review]

**Review by stefashkaa — APPROVED** [review]

**Review by vovac12 — APPROVED** [review]

**Review by bragov4ik — APPROVED** [review]
LGTM, but didn't understand which limit order events are mentioned here

**Review by bragov4ik — APPROVED** [review]
LGTM

File: pallets/order-book/src/lib.rs [hunk 1]
```diff
@@ -321,9 +324,31 @@ pub mod pallet {
         /// Failed to cancel expired order
         ExpirationFailure {
             order_book_id: OrderBookId<AssetIdOf<T>>,
+            dex_id: T::DEXId,
             order_id: T::OrderId,
             error: DispatchError,
         },
+
+        /// The some amount of the limit order is executed
+        LimitOrderExecuted {
```
[the remark this answers is not in the document]
Comment (wer1st) [reply pallets/order-book/src/lib.rs:333]: Do you mean to add it to `LimitOrderExecuted`?
If yes, it is already exists, see line 336

**Comment by wer1st** [comment]
Decided to apply another approach

**Review by vovac12 — APPROVED** [review]

**Pull request merged**